core/shared/src/main/scala/scalaz/zio/TaskR.scala

```diff
@@ -0,0 +1,566 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+/*
+ * PHASE 1 DONE
+ * PHASE 2 DONE
+ * PHASE 3 DONE
+ * PHASE 4 DONE
+ * PHASE 6 DONE
+ */
+
+/*
+ * EXTENDS ZIO_E_Throwable
+ */
+object TaskR {
+
+  type TaskrE = Throwable
```

Comment:
As in the other cases, I'd delete:

1. All the temp comments.
2. The `TaskrE` and `TaskR` type synonyms
3. All the Scaladoc (instead link to ZIO Scaladoc)